site/blog.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>amb-about</title>
+  <link href="../assets/styles/normalize.css" rel="stylesheet" >
+  <link href="../assets/styles/main.css" rel="stylesheet" >
+  <link href="../assets/styles/blog_style.css" rel="stylesheet" >
+</head>
+<body>
+
+  <header>
+    <a href="index.html">
+      <div>
+        <h3>amb</h3>
+      </div>
+    </a>
+    <nav>
+      <ul>
+        <li><a href="portfolio.html"> porotfolio </a></li>
+        <li><a href="blog.html"> blog </a> </li>
+        <li><a href="about.html"> about </a></li>
+        <li><a href="#contact"> contact </a> </li>
+      </ul>
```

Comment:
it's a personal preference kind of thing, but maybe consider removing the link to the page the user is currently on from the header? I've seen it both ways just something to think about since you're not reusing the same header. 
Also, stash away for the future the idea of partials - with rails etc you'll be able to reuse the same code for html consistent to multiple pages. Instead of repeating it like you do here.